.husky/commit-msg

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+yarn commitlint --edit $1
```

Comment:
husky now works based on these `.husky` files, instead of package.json entries.

Kind of annoying, but I want to stay up to date.

Tested all these out with various local commits.